aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -77,6 +77,24 @@ module aptos_std::simple_map {
         vector::push_back(&mut map.data, Element { key, value });
     }
 
+    public fun upsert<Key: store + drop + copy, Value: store + drop + copy>(
+        map: &mut SimpleMap<Key, Value>,
+        key: &Key,
+        value: &Value
+    ) {
+        let len = std::vector::length(&map.data);
```

Comment:
nit: Just vector::length (std:: is not needed)

Reply:
Done